lang/cs/Org.Apache.REEF.Client/API/IJobSubmissionBuilder.cs

```diff
@@ -71,11 +71,16 @@ public interface IJobSubmissionBuilder
         IJobSubmissionBuilder SetJobIdentifier(string id);
 
         /// <summary>
-        /// Set driver memory in megabytes
+        /// Set driver memory in megabytes.
         /// </summary>
         IJobSubmissionBuilder SetDriverMemory(int driverMemoryInMb);
 
         /// <summary>
+        /// Set the maximum amount of times a job can be submitted.
+        /// </summary>
+        IJobSubmissionBuilder SetMaxApplicationSubmissions(int maxAppSubmissions);
```

Comment:
@markusweimer adding this method is a breaking change on this interface. I am not sure about how people use it. Is this fine?